src/34-protect-email.ts

```diff
@@ -6,3 +6,5 @@ export {};
 
 console.log(protectEmail("secret123@codelex.io")); // Expected result: sec...@codelex.io
 console.log(protectEmail("example@example.com")); // Expected result: exa...@example.com
+console.log(protectEmail('12345@example.com')); // Expected result: 12...@example.com
```

Comment:
In terms of the number of characters, this example seems practically the same as the ones we already have.  There's no need to have a separate test case for usernames that consist of numbers only if that's what you were going for.